src/praisonai/praisonai/observability/langextract.py

```
@@ -61,6 +143,46 @@ def __init__(self, config: Optional[LangextractSinkConfig] = None) -> None:
+            )
+            context_emitter = ContextTraceEmitter(
+                sink=sink.context_sink(),
+                session_id=session_id,
+                enabled=True,
+            )
+            set_context_emitter(context_emitter)
+        except ImportError:
+            # Context emitter bridging is optional if not available
+            if warn_callback:
+                warn_callback("ContextTraceEmitter not available")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)